test/AdapterTests.cpp

```diff
@@ -371,27 +371,6 @@ TEST_CASE( "Test destination mode", "[destination]") {
     {
     return (msg.type() == com::amazonaws::iot::securedtunneling::Message_Type_STREAM_RESET) && msg.streamid() == 1;
```

Comment:
is it possible that the reason of this test failure is that we have reset the stream? maybe process did not have enough time between resetting stream and creating new connection.